spec2policy.py

```diff
@@ -159,6 +159,10 @@ def policy_update_flags(change_name = True, change_avatar = True, forbid_create_
     flags_data['flags']['allowCustomUserDisplayNames'] = change_name
     flags_data['flags']['allowCustomUserAvatars'] = change_avatar
     flags_data['flags']['forbidRoomCreation'] = forbid_create_room
+    flags_data['flags']['allowCustomPassthroughUserPasswords'] = False
```

Comment:
A cosmetic remark: even if it is not used below, it is kind of strange to have both values that can be changed when calling the function and values that cannot be changed. I would prefer to have either everything hard-coded or everything customizable.

Reply:
Otherwise it all look good by just looking at the diffs (I just have to upgrade to benefit from spaces ;) ).
A question: are people who are in the _forbidroomcreation_ LDAP groups still able to create direct conversations?